src/components/Panels/SaleInfoPanel/index.tsx

```diff
@@ -85,15 +88,17 @@ export const SaleInfoPanel = () => {
               value: getBalanceString(
                 currentPrice.toString(),
                 decimals,
-                symbol
+                symbol,
+                network === NetworkType.ROCOCO ? 7 : 2
```

Comment:
Hardcoding values like this is inconsistent. If a number is smaller than 1, we should show the first digit and the next two numbers. If greater than 1, we should use 2 decimals only.

Reply:
So 3 decimals for < 1 and 2 decimals for > 1?

Reply:
Yeah, I think that makes sense